Sources/BufferOutputFacility.swift

```diff
@@ -11,32 +11,32 @@ import Foundation
 @objc(RDTBufferOutputFacility) public final class BufferOutputFacility: NSObject, OutputFacility {
 
 	/// A buffer of request representations.
-	public private(set) var requestRepresentations: [RequestRepresentation] = []
+	public fileprivate(set) var requestRepresentations: [RequestRepresentation] = []
```

Comment:
Yes, but I don't see any extensions. 😉 

I just prefer the most restrictive access modifier – if there are no extensions, there is nothing that justifies them to be `fileprivate`.